### PR TITLE
Hck 3801 dhl snowflake add support for stored procedures functions and sequences

### PR DESCRIPTION
**Pull request by WilhelmWesser**
Added support for Procedures RE in SnowFlake

### CODE REVIEW

File: properties_pane/container_level/containerLevelConfig.json [hunk 1]
```diff
@@ -753,6 +753,164 @@ making sure that you maintain a proper JSON format.
...
+		"lowerTab": "Procedures",
+		"structure": [{
+			"propertyName": "Procedure",
+			"propertyType": "group",
+			"propertyKeyword": "procedures",
+			"propertyTooltip": "Creates a new stored procedure or replaces an existing procedure for the current database.",
+			"structure": [
+				{
+					"propertyName": "Name",
+					"propertyKeyword": "name",
```
Comment (WilhelmWesser) [inline properties_pane/container_level/containerLevelConfig.json:767]: Place procedures after functions

File: properties_pane/container_level/containerLevelConfig.json [hunk 2]
```diff
@@ -753,6 +753,164 @@ making sure that you maintain a proper JSON format.
...
+						"Classic Console/SnowSQL"
+					]
+				},
+				{
+					"propertyName": "Comments",
+					"propertyKeyword": "comments",
+					"propertyType": "details",
+					"template": "textarea"
+				}
+			]
```
Comment (WilhelmWesser) [inline properties_pane/container_level/containerLevelConfig.json:912]: change keyword to description
and add Remarks field like in UDFs

Comment (WilhelmWesser) [reply properties_pane/container_level/containerLevelConfig.json:912]: And place comments and remarks like in udfs

File: properties_pane/container_level/containerLevelConfig.json [hunk 3]
```diff
@@ -753,6 +753,164 @@ making sure that you maintain a proper JSON format.
...
+					"propertyKeyword": "language",
+					"propertyTooltip": "language",
+					"propertyType": "select",
+					"defaultValue": "sql",
+					"options": [
+						"sql",
+						"javascript",
+						"java",
+						"python",
+						"scala"
```
Comment (WilhelmWesser) [inline properties_pane/container_level/containerLevelConfig.json:802]: check udfs new languages (if any)

**Review by mtseluiko — APPROVED** [review]

**Pull request merged**